scripts/onboarding/aks/syslogonboarding/existingClusterOnboarding.json

```diff
@@ -0,0 +1,274 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "aksResourceId": {
+            "type": "string",
+            "metadata": {
```

Comment:
We should also include ARM templates for Arc K8s  as well.

Reply:
do we have arm templates for msi dcr onboarding for arc?